src/pks/mpc/CMakeLists.txt

```diff
@@ -41,8 +41,9 @@ set(ats_mpc_src_files
   mpc_surface.cc
   mpc_delegate_water.cc
   mpc_coupled_water.cc
+#  mpc_coupled_dualmedia_water.cc
```

Comment:
remove dead code

Reply:
Ok, so is this MPC meant to be included in this pull request or not?

Reply:
After discussion with Daniil, no it was not intended to be included in this PR.  We will remove this code for PR, then revert it and keep working on it for future PRs.